command/sub8_launch/launch/subsystems/odometry.launch

```diff
@@ -13,11 +13,11 @@
     </rosparam>
     <remap from="odom" to="imu_odom"/>
   </node>
-  <node pkg="nodelet" type="nodelet" name="transform_odometry" args="standalone odometry_utils/transform_odometry"><rosparam>
+  <node pkg="nodelet" type="nodelet" name="transform_odometry" args="standalone odometry_utils/transform_odometry" respawn="true"><rosparam>
```

Comment:
Needed to give time for a variety of things to happen, this has been causing us to lose map frame.